_posts/2015-11-18-Stats-2015.markdown

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title:  "2015"
+author: orta
+categories: stats community 
+---
+
+> CocoaPods is a dependency manager only in the sense that Apple is a cell phone manufacturer – it’s just one of their many projects.
+
+--  Ash Furrow - [Building Online Communities](http://ashfurrow.com/blog/building-online-communities/)
+
+Last year, around this time, I started an [annual write-up](/Stats-2014/) for CocoaPods. A lot happens in a project with 10 relatively active developers, and a lot of very welcome drive-by contributors. 
+
+<!-- more -->
+
+## Contributions
+
+We started the year with a change in ownership. [Eloy Durán](https://twitter.com/alloy) who had been heading up the project for five years, [announced](/The-captain-leaves-the-bridge) he was stepping back due to becoming a father.
+
+Right now we have four people making a lot of the major decisions around CocoaPods: [Boris Bügling](http://buegling.com), [Marius Rackwitz](http://mariusrackwitz.de), [myself](http://orta.io) and [Samuel Giddins](http://segiddins.me). With the help from [other developers](/about) on a day to day basis.
+
+This year we've seen Samuel Giddin's being [sponsored by Capital One](/Capital-One) in order to help him focus on building CocoaPods while he studies at college.
+
+I want to dig in a little about Samuel's work in the last year. In the [last write-up](/Stats-2014/), I mentioned a project he was working on:
+
+> The second, Samuel Giddins, has been mostly working on a iterative dependency resolver for both CocoaPods and Bundler ( CocoaPods for Ruby. )
+
+This project turned into a gem called [Molinillo](https://github.com/CocoaPods/Molinillo/), see the [announcement](/CocoaPods-0.35/), which is the project at CocoaPods with the most impact. Molinillo is now used in both [Bundler](https://github.com/bundler/bundler/pull/3257) and [RubyGems](https://github.com/rubygems/rubygems/pull/1189)! This is awesome because it powers a _huge_ amount of web infrastructure, but that soon enough CocoaPods' code will be shipping in every copy of OS X. So cool.
```

Comment:
Some iOS devs might be unfamiliar with what a "gem" is, so this could be a great opportunity to be super-clichéd.

> This project turned into a gem of a library called ...